docs/dev-proc/standard-release-process.md

```diff
@@ -1,13 +1,38 @@
 # Standard Release Process
 
 Following procedure is generic description of release process of firmware for
-supported hardware platforms. Precise steps and any difference from standard
-process are described in platform specific documentation.
+supported hardware platforms and targets open-source firmware developers.
+Precise steps and any difference from standard process are described in
+platform specific documentation.
 
-Description here is, intentionally, Open Source Firmware framework agnostics
+Description here is, intentionally, open-source firmware framework agnostics
 and should be maintained in that way.
 
-## Process steps
+## Initial release
+
+For platforms not supported by Dasharo there is need for initial release, which
+consist of [Dasharo Common
+Base](../../osf-trivia-list/dasharo/#what-is-dasharo-common-base). Initial
+release should have established test scope defined in platform [test
+matrix](../../unified-test-documentation/overview/#test-matrix) section.
+
+1. Fetch upstream project
+1. Checkout branch `<platfrom>/release`
+    - `coreboot` - use most recent commit on upstream `master`
+    - `edk2` - use most recent stable tag
+1. Create branch `<platform>/rel_v0.1.0`
+1. Cherry-pick required patches from `dasharo-common-base` branch
+1. Apply all other modifications necessary to  provide initial support for
+   `<platform>` according to platform test matrix scope.
+1. Run platform regression test suite
```

Comment:
The steps below assume that platform port is ready. No regression would be run without a port that is somewhat bootable

Reply:
@miczyg1 what change you expect in procedure? Previous point explain that all modifications to provide initial support should be applied to the branch.